deeppavlov/models/preprocessors/capitalization.py

```diff
@@ -84,7 +86,15 @@ def process_word(word: str, to_lower: bool = False,
             ('<FIRST_UPPER>' for first capital and '<ALL_UPPER>' for all caps)
 
     Returns:
-        a preprocessed word
+        a preprocessed word.
+
+    Example:
+        Input params example:
+            word="Zaman",
+            to_lower=True,
+            append_case="first"
+        Output example:
+            ('<FIRST_UPPER>', 'z', 'a', 'm', 'a', 'n')
```

Comment:
A good example of usage examples: https://github.com/deepmipt/DeepPavlov/blob/master/deeppavlov/models/embedders/bow_embedder.py

Reply:
Thanks, updated code according to template